modules/vaos/app/services/vaos/v2/eps/redis_client.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+require 'common/models/concerns/cache_aside'
+
+module VAOS
+  module V2
+    module Eps
+      class RedisClient < Common::RedisStore
```

Comment:
just a general convention, this should represent the data we're getting or saving through RedisStore. So would suggest renaming this to `AccessToken`

Reply:
Addressed in [b2d26f8](https://github.com/department-of-veterans-affairs/vets-api/pull/19307/commits/b2d26f892ec2a4d4b1bf2663b14864b50dce13e6)